tests/decomposition/conftest.py

```diff
@@ -162,7 +161,6 @@ def _crot(*_, **__):
 decompositions["C(PauliX)"] = [_controlled_x_decomp]
 decompositions["C(GlobalPhase)"] = [_controlled_g_phase_decomp]
 decompositions["C(Hadamard)"] = [_controlled_hadamard]
-decompositions["C(PhaseShift)"] = [_controlled_phase_shift_decomp]
```

Comment:
Why we don't need controlled phaseshift any more?

Reply:
This particular decomposition is a duplicate of a more general decomposition rule, so it's removed. Note that this is just in `conftest`.